coral/src/app/pages/not-found/index.tsx

```diff
@@ -15,6 +24,25 @@ const NotFound = () => {
         <Typography.DefaultStrong>
           <Link href={"/index"}>Return to the old interface.</Link>
         </Typography.DefaultStrong>
+
+        <AsyncNativeSelectWrapper
+          entity={"test"}
+          data={data}
+          isLoading={isLoading}
+          isError={isError}
+          error={error}
+          selectElement={
```

Comment:
Will do!

Reply:
I didn't get the typing right for child element, that's why I tried a prop. But it turns out it was not bc. of "children" but the typing is generally hard ^^

Reply:
Hm, it seems TS does not support typing `children`

https://github.com/microsoft/TypeScript/issues/21699?ref=arahansen.com

I also did not yet get a script typing for the prop work though. Not sure if we say Typing > Convenient?  In that case I would see that I get the typing script for a prop, otherwise use `children` (maybe we can add a manual error handing at least)

Reply:
If we want it to type it for the child to be a specific component, I think we can do it with a typeguard throwing runtime errors like this? https://codesandbox.io/p/sandbox/react-typescript-forked-g46s3c?file=%2Fsrc%2FApp.tsx%3A64%2C1

Reply:
Yes, typeguard was what I meant with "(maybe we can add a manual error handing at least)" (not sure why I made it so complicated ^^). 

I'll see if I get that working in a reasonable time and otherwise make sure it can only be a ReactNode.

Reply:
Ah sorry I didn't catch your meaning ^^'

Reply:
>Ah sorry I didn't catch your meaning ^^'

I mean, my wording was too complicated :D